.internal-ci/helm/fog-ingest/templates/supervisord-fog-ingest-configmap.yaml

```diff
@@ -2,26 +2,15 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "fogIngest.fullname" . }}-supervisord-fog-ingest
+  name: {{ include "fog-ingest.fullname" . }}-supervisord-fog-ingest
   labels:
-    {{- include "fogIngest.labels" . | nindent 4 }}
+    {{- include "fog-ingest.labels" . | nindent 4 }}
 data:
   ingest.conf: |
     [program:fog-ingest]
     priority=100
-    environment=MC_SENTRY_DSN="%(ENV_FOG_INGEST_SENTRY_DSN)s"
     command=/usr/bin/fog_ingest_server
-      --pubkey-expiry-window %(ENV_FOG_PUBKEY_EXPIRY_WINDOW)s
-      --peers {{ include "fogIngest.peerURLs" . }}
-      --local-node-id %(ENV_LOCAL_NODE_ID)s
-      --ias-spid %(ENV_IAS_SPID)s
-      --ias-api-key %(ENV_IAS_API_KEY)s
-      --ledger-db /fog-data/ledger
-      --watcher-db /fog-data/watcher
-      --client-listen-uri insecure-fog-ingest://0.0.0.0:3226/
-      --peer-listen-uri insecure-igp://0.0.0.0:8090/
-      --state-file /ingest-run-data/ingest-state-file
-      --admin-listen-uri insecure-mca://127.0.0.1:8001/
+      --peers {{ include "fog-ingest.peerURLs" . }}
```

Comment:
Move most of the configuration into env vars. Peers needs to be defined on the command line since MC_PEERS is also used for a different purpose in mobilecoind.